db/migrate/20210624113422_add_type_to_transient_registrations.rb

```diff
@@ -0,0 +1,14 @@
+class AddTypeToTransientRegistrations < ActiveRecord::Migration[6.0]
+  def up
+    add_column :transient_registrations, :type, :string, null: false
+    # Set any existing TransientRegistrations to NewRegistrations so everything has a type
+    execute <<-SQL
+       UPDATE transient_registrations
+       SET type = 'FloodRiskEngine::NewRegistration';
+    SQL
+  end
```

Comment:
This didn't work for me locally, as the type column doesn't allow nulls. Perhaps add the column with a default value, then add the `null:false` afterwards?

Reply:
This could be a development only issue as the transient_registrations table is new.

Reply:
I've had a go at setting up a default first and then adding null later – how's this work for you? https://github.com/DEFRA/flood-risk-engine/pull/393/commits/e527647807d636d4752d0c1569ddc9c1aaaf70d2

Reply:
Looks good! Is there an index on the `type`?

Reply:
There isn't in WEX so I didn't add one here, but might be a good idea to add it